tests/aave/ethereum/aave_borrow_test.sql

```diff
@@ -0,0 +1,25 @@
+with unit_test1
+    as (select case
+                 when usd_amount = 499785 then true
+                 else false
+               end as test
+        from   {{ ref('aave_v2_ethereum_borrow' )}}
+        where  token_address = '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'
+               and borrower = '0xec46dd165ee2d4af460a9c3d01b5a4c9516c9c3f'
+               and evt_block_time = '2022-09-22 10:45'),
```

Comment:
is it easy to specify the transaction hash here?

Reply:
Added